addon/components/listing.hbs

```diff
@@ -1,32 +1,98 @@
-<div class="au-o-grid">
-  <div class="au-o-grid__item au-u-4-5">
-    {{#each this.subForms as |subForm index|}}
-      <SubForm
-        @subForm={{subForm}}
-        @formStore={{@formStore}}
-        @graphs={{@graphs}}
-        @sourceNode={{subForm.sourceNode}}
-        @forceShowErrors={{@forceShowErrors}}
-        @cacheConditionals={{@cacheConditionals}}
-        @last={{eq index (sub this.subForms.length 1)}}
-        @show={{@show}}
-        @level={{@level}}
-        @onRemoveEntry={{this.removeEntry}}
-        @canRemove={{@listing.canRemove}}
-      />
-    {{/each}}
-  </div>
-  {{#if this.canAdd}}
-    <div class="au-o-grid__item au-u-1-5">
+{{#if @listing.isTable}}
+  <div>
+    <AuTable class="listing-table">
+      <:title>
+        {{! TODO: read this from the graph directly }}
+        {{this.subForms.firstObject.itemLabel}}
+      </:title>
+      <:header>
+        {{!TODO this can probably be done without the subforms setup by simply retrieving the labels from the graph in some way?}}
+        {{#let this.subForms.firstObject as |subForm|}}
```

Comment:
We need a list of al the labels of the form fields that belong to this listing. I've used the SubForm component as a hacky solution for now, but there must be better ways.

Reply:
This is an argument for an extra headers component defined in the form.ttl (as long as it is transparent for the backend)